gstcefsrc.cc

```diff
@@ -680,6 +696,11 @@ gst_cef_src_class_init (GstCefSrcClass * klass)
           "The URL to display",
           DEFAULT_URL, (GParamFlags) (G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS | G_PARAM_CONSTRUCT)));
 
+  g_object_class_install_property (gobject_class, PROP_GPU,
+    g_param_spec_boolean ("gpu", "gpu",
+          "Enable GPU",
```

Comment:
Please expand a bit on the description :)

Reply:
I added a bit more information, but if you want it to tell something else, let me know ;)